javascript/ql/src/Security/CWE-834/TaintedLength.qhelp

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+<overview>
+	<p>
+		Using the .length property of an untrusted object as a loop bound may
+		cause indefinite looping since a malicious attacker can set the
+		<code>.length</code> property to a very large number. For example, 
+		when a program that expects an array is passed a JSON object such as
+		<code>{length: 1e100}</code>, the loop will be run for 1e100 
```

Comment:
I'd slightly prefer "10<sup>100</sup>" over "1e100" for clarity. Unless you have a particular reason to prefer the latter?

Reply:
I didn't know <sup> was a thing I could use. I changed it.